src/assets/scss/components/_quoteblock.scss

```diff
@@ -6,21 +6,26 @@
     
     // Different padding for quote block with images
     &.quoteblock--hasimage {
-        padding-top: 7rem;
-        padding-bottom: 7rem;
+        padding-top: 6rem;
+        padding-bottom: 8rem;
 
         @media #{$medium-up} {
-            padding-top: 8rem;
-            padding-bottom: 8rem;
+            padding-top: 7rem;
+            padding-bottom: 9rem;
         }
 
         @media #{$large-up} {
-            padding-top: 12rem;
-            padding-bottom: 11rem;
+            padding-top: 10rem;
+            padding-bottom: 12rem;
         }
     }
 }
 
 .quoteblock__image {
     @include image-rounded($secondary-color);
+    margin: auto auto 1.5rem;
```

Comment:
@alxmrtnz this is essentially `margin-bottom` right? we can be explicit and just say `margin-bottom: 1.5rem` if that's the case, better to be clear.